.ci/scripts/test_export_data_command.sh

```diff
@@ -2,29 +2,23 @@
 
 # Test for the export-data admin command against sqlite and postgres
 
+# Expects Synapse to have been already installed with `poetry install -e .[postgres]
```

Comment:
Should this be?

```suggestion
# Expects Synapse to have been already installed with `poetry install --extras postgres`.
# Expects `poetry` to be available on the `PATH`.
```

From #12412

Reply:
It definitely should be, thank you. I suppose my brain was frazzled into confusing `poetry` and `pip` at some point.